.github/workflows/build-release.yml

```diff
@@ -0,0 +1,46 @@
+name: build
+on:
+  pull_request:
+  # Run builds nightly to catch incompatibilities with new marshmallow releases
+  schedule:
+  - cron: "0 0 * * *"
+jobs:
+  lint:
```

Comment:
should we use pre-commit.ci like in webargs?

Reply:
We could. TBH, I followed the webargs thread from afar. This PR is based on the setup I use in for flask-smorest so it was easy for me to adapt.

We can use pre-commit.ci in a later phase. I'd like to merge this sooner to fix CI tests asap.

Reply:
sounds good. my comments are not blocking, so go ahead and merge when you feel it's right